Meadowlark-js/backends/meadowlark-kafka-stream/docker/docker-compose.yml

```diff
@@ -31,11 +31,10 @@ services:
       - ZOOKEEPER_CONNECT=zookeeper:2181
       - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka1:9092
 
-
-  connect:
-    hostname: kafka-connect
-    container_name: kafka-connect
-    image: edfialliance/connect-meadowlark:2.3-1@sha256:6605d2f0ad1797ccf7e3f7a4dbe690bb0c9e198dd6a0d5720a7b170d0bc4ca95
+  connect-source:
+    hostname: kafka-connect-source
+    container_name: kafka-connect-source
+    image: edfialliance/connect-meadowlark:2.3-2@sha256:6605d2f0ad1797ccf7e3f7a4dbe690bb0c9e198dd6a0d5720a7b170d0bc4ca95
```

Comment:
Sha256 should be sha256:e6792ab9c797a27ae15e2fba68e874cf9898658f947ba0448182b5e269b1bbb8 see https://hub.docker.com/layers/edfialliance/connect-meadowlark/2.3-2/images/sha256-e6792ab9c797a27ae15e2fba68e874cf9898658f947ba0448182b5e269b1bbb8?context=explore